strr-examiner-web/app/components/HostDetailsView.vue

```diff
@@ -22,25 +23,42 @@ const openDocInNewTab = async (supportingDocument: ApiDocument) => {
   URL.revokeObjectURL(url)
 }
 
+const businessLicenceDoc = registration.documents
+  .find(doc => doc.documentType === DocumentUploadType.LOCAL_GOVT_BUSINESS_LICENSE)
+
+// Principal Residence Requirements:
+const getPrRequired = (): string =>
+  strRequirements?.isPrincipalResidenceRequired
+    ? t('pr.required')
+    : t('pr.notRequired')
+
+const getPrExemptReason = (): string =>
+  registration.unitDetails.prExemptReason
+    ? t(`prExemptReason.${registration.unitDetails.prExemptReason}`)
+    : t('prExemptReason.notExempt')
+
+const getOwnershipType = (): string =>
+  registration?.unitDetails.ownershipType === OwnershipType.RENT ? `${t('ownershipType.RENT')}.` : ''
```

Comment:
Just checking -- you only want to show the ownership type if it is RENT ?

Reply:
Yea, from the Requirements it says to show Renter, and all the docs for all types.